README.md

```diff
@@ -318,6 +318,45 @@ use a combination of `offset` and `limit` parameters to paginate results.
 To be used when the result window is superior to `10 000`, allows to easily loop across all pages of a query response.
 use `limit` and `offset` along with `search_after` filled with the value from the `X-Sort` response header to get the next page.
 
+#### Rate limit
+
+Requests may be rate limited by enabling the middleware using the `ctia.http.rate-limit.enabled` property.
+
+It rate limits how many HTTP requests a CTIA group can make in an hour. The group is identified with the property :identity of the current Ring request.
+
+Before the rate limit is reached, the header `X-Ratelimit-Group-Limit` is returned in the response:
```

Comment:
I would also specify that `X-RateLimit-GROUP-Remaining` is also returned.

Reply:
`X-RateLimit-GROUP-Remaining` is not returned